faults.csv

```diff
@@ -1,63 +1,63 @@
-,USH,Upstream HOM temperature is too high,Upstream HOM temperature is too high (could indicate an impending quench),CRYO,EDM,$EDM/cryo/cryo_system_all.edl,,,CTE:CM{CRYOMODULE}:1{CAVITY}18:UH:,TEMP.SEVR,,2,2,Upstream HOM temperature is too high
-,DSH,Downstream HOM temperature too high,Downstream HOM temperature too high (could indicate an impending quench),CRYO,EDM,$EDM/cryo/cryo_system_all.edl,,,CTE:CM{CRYOMODULE}:1{CAVITY}20:DH:,TEMP.SEVR,,2,2,Downstream HOM temperature too high
-,AOT,Amplitude out of tolerance,The amplitude readback is too far away from the desired amplitude,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,SELCHAR=3,,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,AACTMEAN.SEVR,0,,2,Amplitude out of tolerance
-,POT,Phase out of tolerance,The phase readback is too far away from the desired phase,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=5,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,PACTMEAN.SEVR,0,,2,Phase out of tolerance
-,QBP,Quench Interlock Bypassed,Quench interlock bypassed,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=4,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,QUENCH_BYP_RBV,,1,1,Quench Interlock Bypassed
-,AFB,Feedback Amplitude out of tolerance,The feedback maintaining the amplitude is unable to keep the amplitude within tolerance. (Must be checked after RFREADYFORBEAM). Amplitude feedback of measured gradient is too low,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=5,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,AMPFB_SUM,0,,1,Feedback Amplitude out of tolerance
-,PFB,Phase feedback out of tolerance,The feedback maintaining the phase is unable to keep the phase within tolerance. (Must be checked after RFREADYFORBEAM),CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=5,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,PHAFB_SUM,0,,1,Phase feedback out of tolerance
+﻿Three Letter Code,Short Description,Long Description,Recommended Corrective Actions,Level,Button Type,Button Path,Button Macros,Rack,PV Prefix,PV Suffix,OK If Equal To,Faulted If Equal To,Severity,Generic Short Description for Decoder
+OFF,Offline,Cavity not usable or not intended to be used for extended period,No further action required,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=10,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,HWMODE,,2,5,Offline
+MNT,Maintenance,Maintenance: Cavity is known to be bad and is actively being worked on,No further action required,CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=10,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,HWMODE,,1,4,Maintenance
+MND,Maintenance Done,Maintenance done but cavity has not been run up for validation,"If program permits, set online and verify stability. If stable, leave online. If unstable, write a relevant CATER and set offline",CAV,EDM,$EDM/llrf/rf_srf_cavity_main.edl,"SELTAB=10,SELCHAR=3",,ACCL:{LINAC}:{CRYOMODULE}{CAVITY}0:,HWMODE,,3,4,Maintenance Done

```

Comment:
Since there should already be a relevant cater, how about '... If unstable, comment on existing CATER and set offline'?